MARIE_AND_BERT/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM tiangolo/uwsgi-nginx-flask:python3.8 as flask
+ENV UWSGI_CHEAPER 0
+ENV UWSGI_PROCESSES 1
+WORKDIR /app
+
+COPY Marie/ ./Marie
+COPY uwsgi.ini ./uwsgi.ini
+COPY static/ ./static
+COPY templates/ ./templates
+COPY main.py ./main.py
+COPY requirements_linux.txt ./requirements_linux.txt
+COPY DATA/ ./DATA
+COPY DATA/label_dict.js ./static/js/label_dict.js
```

Comment:
This is inconsistent with the instructions in the main readme file, where it says to put the `label_dict.js` directly into `./static/js`. Please fix this.